big_tests/tests/rest_helper.erl

```diff
@@ -164,7 +164,7 @@ decode(RespBody) ->
     try
         jiffy:decode(RespBody)
     catch
-        throw:_ ->
+        error:_ ->
```

Comment:
Really? Is this the reason why REST tests were failing after `jiffy` upgrade? :D

Reply:
yes, they changed the way they raise errors in decode